frontend/src/pages/Home.jsx

```diff
@@ -1,9 +1,9 @@
 function Home() {
   return (
     <img
-      src="/src/assets/images/HeroCorporation-logo-22.png"
-      alt="Hero Corporation Logo"
-      className="HeroCorpLogo"
+      src="/src/assets/images/hero-corporation-logo-mobile.png"
```

Comment:
you're giving an url here (wrapped with quotes), but the value is a path. either give a real url or (better) import the image and give it like a variable